nntrainer/layers/common_properties.cpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   common_properties.cpp
+ * @date   14 May 2021
+ * @brief  This file contains implementation of common properties widely used
+ * across layers
+ * @see	   https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+#include <common_properties.h>
+
+#include <nntrainer_error.h>
+#include <parse_util.h>
+
+#include <regex>
+#include <vector>
+
+namespace nntrainer {
+namespace props {
+bool Name::isValid(const std::string &v) const {
+  static std::regex violation(".*[,+ ].*");
```

Comment:
I wonder if it is better to check alphanumeric (std::alnum()) + allowed connectors(`_`, `-`), than checking for outliers (which can be extended to all ascii characters other than allowed ones)?

Reply:
yeah, let's do that.

Reply:
Done, added connector `/`, I think some ensureName is using this